talli-app/src/components/OrganizerView/NewEventForm.js

```diff
@@ -3,29 +3,50 @@ import { Typography } from '@material-ui/core';
 import '../component_style/NewEventForm.css';
 
 export default class NewEventForm extends React.Component {
+    state = {
+        automateTime: false
+    }
+
+    toggleTime = () => {
+        this.setState({
+            automateTime: !this.state.automateTime
+        });
+    }
+
+    AddEntries() { 
+        this.props.handler(this.props.orgViews.ADD);
+        /* unimplemented */
+    }
+
     render() {
         /**
          * TODO: Cleanup this div and replace <input/>'s
          */
         return (
              <div className='newEventForm'>
                 <Typography variant='h4' align='center' gutterBottom>Create a new event</Typography>
-                <form className='eventForm'>
+                <form className='eventForm' onSubmit={() => this.AddEntries()}>
```

Comment:
This is another big spot where switching to Material-UI components would make the form look more like the rest of the app (and also play nicer with react)